python/options.py

```diff
@@ -0,0 +1,50 @@
+parser.add_argument('--CanvMarginLf',action="store",dest="CanvMarginLf",type=str,default="0.16",help="Set Canvas Margin Left (default is 0.16)") 	
+parser.add_argument('--CanvMarginRt',action="store",dest="CanvMarginRt",type=str,default="0.06",help="Set Canvas Margin Right (default is 0.06)") 
+parser.add_argument('--CanvPlotType',action="store",dest="CanvPlotType",type=str,default="TGraphErrors",help="Set the Canvas Plot type (default is TGraphErrors)") 
+parser.add_argument('--CanvTitleX',action="store",dest="CanvTitleX",type=str,default="Divider Current #left(#muA#right)",help="Set Xaxis title, default is: Divider Current #left(#muA#right)")
+parser.add_argument('--CanvTitleY',action="store",dest="CanvTitleY",type=str,default="Applied Voltage #left(kV#right)",help="Set Yaxis title, default is: Applied Voltage #left(kV#right)")
+parser.add_argument('--CanvRangeX',action="store",dest="CanvRangeX",type=str,default="0,1000",help="Set the X-Range of the Canvas (default is 0,1000)") 
+parser.add_argument('--CanvRangeY',action="store",dest="CanvRangeY",type=str,default="0,7",help="Set the Y-Range of the Canvas (default is 0,7)") 
+parser.add_argument('--CanvTitleOffsetX',action="store",dest="CanvTitleOffsetX",type=str,default="1.0",help="Set the X-Offset of the Canvas Title (default is 1.0)") 
+parser.add_argument('--CanvTitleOffsetY',action="store",dest="CanvTitleOffsetY",type=str,default="0.8",help="Set the Y-offset of the Canvas Title (default is 0.8)") 
+parser.add_argument('--CanvName',action="store",dest="CanvName",type=str,default="LS2_Detectors",help="Set Canvas Name, Default is: LS2_Detectors")
+parser.add_argument('--YaxisScale',dest="YaxisScale",action="store",type=bool, default=False,help="If YaxisScale option is True the Y axis is plotted in kUnit. Default is: False")
+parser.add_argument('--SetErrX',action="store",dest="SetErrX",type=bool,default=False,help="Set ErrX option True or False , Default is: False")
+parser.add_argument('--SetErrY',action="store",dest="SetErrY",type=bool,default=False,help="Set ErrY option True or False , Default is: False")
+parser.add_argument('--SelectColumnErrX',action="store",dest="SelectColumnErrX",type=int,default=0,help="If SetErrX=True choose the Column for XError, Default is: 0")
+parser.add_argument('--SelectColumnErrY',action="store",dest="SelectColumnErrY",type=int,default=0,help="If SetErrY=True choose the Column for YError, Default is: 0")
+parser.add_argument('--PlotLineSize',action="store",dest="PlotLineSize",type=str,default="1",help="Set Plot Line Size (default is: 1)")
+parser.add_argument('--PlotLineStyle',action="store",dest="PlotLineStyle",type=str,default="1",help="Set Plot Line Style (default is: 1)")
+parser.add_argument('--PlotMarkerSize',action="store",dest="PlotMarkerSize",type=str,default="0.8",help="Set Plot Marker Size (default is: 0.8)")
+
+
+
```

Comment:
White space reduction.

Additionally you have declared the type on almost all of these variables here.  Then elsewhere in the PR you spend time (CPU) type-casting them again into the type that they already are!

This wastes CPU time and memory...remove the unnecessary type castings everywhere in the PR